src/pages/developers/tutorials.js

```diff
@@ -189,8 +189,10 @@ const ModalTitle = styled.h2`
 const TutorialsPage = ({ data }) => {
   const intl = useIntl()
 
-  const allTutorials = data.allTutorials.nodes.map((tutorial) => {
-    return {
+  // Filter tutorials by language and map to object
+  const allTutorials = data.allTutorials.nodes
+    .filter((tutorial) => tutorial.frontmatter.lang === intl.locale)
```

Comment:
I think the issue here is that `tutorial.frontmatter.lang` is still `en` for the English pages we copy over.

In gatsby-node.js you can see we don't overwrite that frontmatter prop: https://github.com/ethereum/ethereum-org-website/blob/dev/gatsby-node.js#L145
Instead we add the `language` property in the `intl` object.

Let's update this filter to use that (I believe you can grab it from the `pageContext` prop.